Doc/library/asyncio-dev.rst

```diff
@@ -122,7 +123,7 @@ To schedule a coroutine object from a different thread, the
      future = asyncio.run_coroutine_threadsafe(coro_func(), loop)
      result = future.result(timeout)  # Wait for the result with a timeout
 
-The :meth:`AbstractEventLoop.run_in_executor` method can be used with a thread pool
+The :meth:`loop.run_in_executor` method can be used with a thread pool
```

Comment:
`ThreadPoolExecutor` to execute a callback in a different thread so as not to block the event loop's main thread.